doc/user-guide/13-tcg-opal-support.adoc

```diff
@@ -19,6 +19,83 @@ https://github.com/Drive-Trust-Alliance/exec/blob/master/sedutil_LINUX.tgz?raw=t
 (check version compatibility), or see
 <<How to Build sedutil-cli Version {sedutil-cli-version}>>.
 
+== Quick Start: Setting up Self-Encrypting Disks
+
+NOTE: This section assumes that ReaR was installed from a package repository. If you are using ReaR locally from GitHub: `cd` into the project's base directory, use the configuration file `etc/rear/local.conf`, invoke `usr/sbin/rear`.
+
+=== Step 1: Install Prerequisites
+[arabic]
+. Download the low-level utility `sedutil-cli` (version 1.15.1) from https://github.com/Drive-Trust-Alliance/exec/blob/master/sedutil_LINUX.tgz?raw=true[Drive-Trust-Alliance on GitHub].
+. Install it into a directory within root’s search path (e.g. `/usr/local/sbin`).
+
+=== Step 2: Create Disk Images for PBA and Rescue System
+[arabic]
+. In `/etc/rear/local.conf` add these two lines:
++
+[source,bash]
+----
+OUTPUT=RAWDISK
+OUTPUT_URL="file:///var/lib/rear/output"
+----
+** To support secure boot, add another line specifying your secure boot loader, e.g. on Ubuntu:
++
+[source,bash]
+----
+SECURE_BOOT_BOOTLOADER="/boot/efi/EFI/ubuntu/shimx64.efi"
+----
+. Run `sudo rear mkopalpba` (ignore messages about keyboard mappings)
+. Run `sudo rear mkrescue` (ignore messages about keyboard mappings)
+
+=== Step 3: Create a Bootable USB Stick with the Rescue System
+[arabic]
+. Plug in the USB stick to use.
+. Use `lsblk -o +MODEL` to find the correct disk device path for your USB stick (typically `/dev/sdX`).
+. *RE-CHECK. The following command will overwrite the entire drive.*
```

Comment:
Because you are speaking about a 'USB stick' I would stick to that name
and write "The following command will overwrite the entire USB stick.".
Alternatively you may use "USB drive" or "USB disk" consistently
if you like to point out that a USB mass storage device is a disk(drive).